src/js/views/globals/app-nav/app-nav_views.js

```diff
@@ -31,34 +31,61 @@ const MainNavDroplist = Droplist.extend({
         <img class="app-nav__header-logo" src="/rwell-logo.svg" />
       {{else}}
         <div class="u-text--overflow">
-          <h2 class="app-nav__header-title u-text--overflow">{{ orgName }}</h2>
+          <h2 class="app-nav__header-title u-text--overflow">{{ workspaceName }}</h2>
           <span class="app-nav__header-arrow">{{far "angle-down"}}</span>
         </div>
         <div class="u-text--overflow">{{ userName }}</div>
       {{/if}}
     `,
     templateContext() {
       const currentUser = Radio.request('bootstrap', 'currentUser');
-      const currentOrg = Radio.request('bootstrap', 'organization');
+      const currentWorkspace = Radio.request('bootstrap', 'currentWorkspace');
 
       return {
         userName: currentUser.get('name'),
-        orgName: currentOrg.get('name'),
+        workspaceName: currentWorkspace.get('name'),
         isMinimized: this.getOption('state').get('isMinimized'),
       };
     },
   },
   picklistOptions() {
     return {
       className: 'picklist app-nav__picklist',
-      itemClassName: 'flex flex-align-center app-nav__picklist-item',
-      lists: [{
-        collection: this.collection,
-        itemTemplate: hbs`
-          {{fa icon.type icon.icon classes=icon.classes~}}
-          <span>{{formatMessage text}}</span>
-        `,
-      }],
+      template: hbs`
+        <div class="app-nav__picklist-heading">{{ @intl.globals.appNav.appNavViews.mainNavDroplist.organizationHeading }}</div>
+        <div class="app-nav__picklist-workspace-name">{{ name }}</div>
+        <ul class="flex-region picklist__scroll js-picklist-scroll"></ul>
+        <div class="app-nav__picklist-bottom">
+          <div class="picklist__item app-nav__picklist-item js-help">
+            {{far "life-ring"}}<span>{{ @intl.globals.appNav.appNavViews.mainNav.help }}</span>
+          </div>
+          <div class="picklist__item app-nav__picklist-item js-sign-out">
+            {{fas "right-from-bracket"}}<span>{{ @intl.globals.appNav.appNavViews.mainNav.signOut }}</span>
+          </div>
+        </div>
+      `,
+      templateContext() {
+        const currentOrg = Radio.request('bootstrap', 'organization');
+
+        return {
+          name: currentOrg.get('name'),
+        };
+      },
+      itemClassName: 'app-nav__picklist-item',
+      triggers: {
+        'click @ui.help': 'help',
+        'click @ui.signOut': 'signOut',
+      },
+      ui: {
+        help: '.js-help',
+        signOut: '.js-sign-out',
+      },
+      onHelp() {
+        window.open('https://help.roundingwell.com/');
+      },
+      onSignOut() {
+        Radio.request('auth', 'logout');
+      },
```

Comment:
`onHelp()` and `onSignOut()` are not working... any ideas?

Only way I can get these to call is if I add them directly to the picklist (`/js/components/picklist/index.js` file).

Reply:
This is the only list of options you can pass at instantiation: https://github.com/marionettejs/backbone.marionette/blob/master/src/view.js#L12

So anything else'd have to be `this.listenTo(view, 'help'...` elsewhere

But the droplist can have a `picklistEvents` like the `picklistOptions` which'd be something like:
```
picklistEvents: { 
  'help'() {
    window.open('https://help.roundingwell.com/');
  },
  'signOut'() {
    Radio.request('auth', 'logout');
  },
},
```

or the value can be a function name on the droplist

Reply:
Nice this works. Updated to use this in the last push. So those droplist items work properly now.